Adding new concepts and procedures for Multi CV hosts and AKs (#3609)
theforeman/foreman-documentation: guides/common/modules/con_content-view-environment-categories.adoc

> [id="Content_view_environment_categories_{context}"]

Reviewer: ```suggestion
[id="content-view-environment-categories"]
```

same as above.

> = Content view enviornment categories

Reviewer 2: ```suggestion
= Content view environment categories
```

The content view environments available in your organization fall into the following categories:

> . *Library environment*:

Reviewer: same as above.

> The system assigns the Library environment to represent the Library lifecycle environment and the *Default Organization View* content view.

Reviewer: ```suggestion
{Project} assigns the Library environment to represent the Library lifecycle environment and the *Default Organization View* content view.
```
suggestion/question: clarify wording.

Is that true? If not, what do you mean by "system"?

Reviewer 2: technical answer: Katello code tells Candlepin what the environment name should be :)

{Project} sounds good to me

Unpromoted content views::

> For each content view that you publish but do not promote to a lifecycle environment, the system creates a content view environment.

Reviewer: ```suggestion
For each content view that you publish but do not promote to a lifecycle environment, {Project} creates a content view environment.
```

same as above.

> The system labels these environments using the format: `Library/<content_view_label>`.

Reviewer: ```suggestion
{Project} labels these environments using the format: `Library/<content_view_label>`.
```

same as above.

Promoted content views::

> For each content view that you promote to a specific lifecycle environment, the system creates a corresponding content view environment.

Reviewer: ```suggestion
For each content view that you promote to a specific lifecycle environment, {Project} creates a corresponding content view environment.
```

> The system labels these environments using the format: `<lifecycle_environment_label>/<content_view_label>`.

Reviewer: ```suggestion
{Project} labels these environments using the format: `<lifecycle_environment_label>/<content_view_label>`.
```
